peripherals/hrtim/common.yaml

```diff
@@ -170,6 +170,29 @@ EECR1:
     Src2: [1, "Source 2"]
     Src3: [2, "Source 3"]
     Src4: [3, "Source 4"]
+EECR3:
+  EEVSD:
+    Div1: [0, "f_EEVS=f_HRTIM"]
+    Div2: [1, "f_EEVS=f_HRTIM/2"]
+    Div4: [2, "f_EEVS=f_HRTIM/4"]
+    Div8: [3, "f_EEVS=f_HRTIM/8"]
+  "EE*F":
+    Disabled: [0, "Filter disabled"]
+    Div1_N2: [1, "f_SAMPLING=f_HRTIM, N=2"]
+    Div1_N4: [2, "f_SAMPLING=f_HRTIM, N=4"]
+    Div1_N8: [3, "f_SAMPLING=f_HRTIM, N=8"]
+    Div2_N6: [4, "f_SAMPLING=f_EEVS/2, N=6"]
```

Comment:
According to the RM's only the first settings use f_HRTIM as clock source. The rest use f_EEVS. I have updated this in this PR